sunbeam-python/sunbeam/provider/local/commands.py

```diff
@@ -677,3 +695,126 @@ def remove(ctx: click.Context, name: str, force: bool) -> None:
         f"Run command 'sudo /sbin/remove-juju-services' on node {name} "
         "to reuse the machine."
     )
+
+
+@click.command("deployment")
+@click.option("-a", "--accept-defaults", help="Accept all defaults.", is_flag=True)
+@click.option(
+    "-m",
+    "--manifest",
+    help="Manifest file.",
+    type=click.Path(exists=True, dir_okay=False, path_type=Path),
+)
+@click.option(
+    "-o",
+    "--openrc",
+    help="Output file for cloud access details.",
+    type=click.Path(dir_okay=False, path_type=Path),
+)
+@click.pass_context
+def configure_cmd(
+    ctx: click.Context,
+    openrc: Path | None = None,
+    manifest: Path | None = None,
+    accept_defaults: bool = False,
+) -> None:
+    deployment: Deployment = ctx.obj
+    client = deployment.get_client()
+    preflight_checks = []
+    preflight_checks.append(DaemonGroupCheck())
+    preflight_checks.append(VerifyBootstrappedCheck(client))
+    run_preflight_checks(preflight_checks, console)
+
+    # Validate manifest file
+    manifest_obj = None
+    if manifest:
+        manifest_obj = Manifest.load(
+            deployment, manifest_file=manifest, include_defaults=True
+        )
+    else:
+        manifest_obj = Manifest.load_latest_from_clusterdb(
+            deployment, include_defaults=True
+        )
+
+    LOG.debug(
+        f"Manifest used for deployment - preseed: {manifest_obj.deployment_config}"
+    )
+    LOG.debug(
+        f"Manifest used for deployment - software: {manifest_obj.software_config}"
+    )
+    preseed = manifest_obj.deployment_config or {}
+
+    name = utils.get_fqdn()
+    tfplan = "demo-setup"
+    tfplan_dir: str = TERRAFORM_DIR_NAMES.get(tfplan)
+    snap = Snap()
+    manifest_tfplans = manifest_obj.software_config.terraform
```

Comment:
Can we use manifest.get_tfhelper()

probably L#755 should be moved to get_tfhelper function.